build/tools/Automation/Services/HttpService.cs

```diff
@@ -33,7 +33,6 @@ public async Task<string> GetDataAsync(string url)
         {
             try
             {
-                Console.WriteLine("Making GET request to: " + url);
                 HttpResponseMessage response = await this.httpClient.GetAsync(url);
                 Console.WriteLine($"Response received: {response.StatusCode}");
```

Comment:
Should we remove this line also?

Reply:
I think we can leave this on as it just outputs: 
![image](https://github.com/microsoft/Oryx/assets/86327553/2b3004a8-77e2-4bf8-bfd5-e0ad450669dc)

The reason I removed [line 36](https://github.com/microsoft/Oryx/pull/2072/files#diff-1f46b896008cc1f309d5d61a8c7268ce4d5abb3b683377f4ed0f536e9c278ecdL36) is because we're adding the SAS token to the URL and I don't want that to leak in our logs.

Reply:
Makes sense!

Reply:
Yes we can change that. Would it be ok if we moved logging for another time? Since this update is needed to unblock Automation from releasing the new .NET SDKs.

Reply:
Sounds good to me!